recipes/find-kedro/meta.yml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please delete all comments from the example `meta.yaml`

Reply:
I have removed comments